src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
```

Comment:
U have a lot of double empty rows, pls make formating according to best practices
https://google.github.io/styleguide/javaguide.html#s4-formatting 
If u use Intelij idea u can use on Win machine combination `Ctrl+Alt+L`, for other need to google